pyproject.toml

```
@@ -10,7 +10,7 @@ include = [
 
 [project]
 name = "pyhf_combine_converter"
-version = "0.0.2"
+version = "0.0.3"
```

```suggestion
version = "0.0.2"
```

I'd hold off on bumping version until you know that you have a new release that you're ready to put out. Once things are tested and merged then it is common to have a commit that does nothing other than bump the version.